docs/pages/tutorial/deploy-to-mainnet/main.md

```diff
@@ -0,0 +1,172 @@
+## Sepolia / Mainnet Deployment
+
+This is a guide to deploy a Dojo world to Sepolia.
+The steps for Mainnet are exactly the same, just replace the chain name and ID when necessary.
+
+
+### Setup
+
+* You need a [Starknet RPC Provider](https://www.starknet.io/fullnodes-rpc-services/) to deploy contracts on-chain. After you get yours, check if it works and is on the chain you want to deploy (`SN_SEPOLIA` or `SN_MAIN`)
+
+```sh
+# run this...
+curl --location '<RPC_PROVIDER_URL>' \
+--header 'Content-Type: application/json' \
+--data '{"id": 0,"jsonrpc": "2.0","method": "starknet_chainId","params": {}}'
```

Comment:
Why not using starkli here? To avoid installing the tool?

Reply:
I think this is pretty nice as it works out of the box for most devs. Only requiring curl and xxd.

At least I already had those installed but not starkli. So I think it's more friendly for new starknet devs. 

Open to using starkli if you still think it's the more accessible move.